src/web/services/log.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { LogTypes, ResourceEndpoints } from '../types/api-const';
+import { HttpRequestService } from './http-request.service';
+
+/**
+ * Handles logging related logic provision.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class LogService {
+
+  constructor(private httpRequestService: HttpRequestService) { }
+
+  /**
+   * Creates a log for feedback session by calling API.
+   */
+  createFeedbackSessionLog(queryParams: {
+    courseId: string,
+    studentEmail: string,
```

Comment:
same here. log down the fsname too

Reply:
I was also thinking of logging the `fsname` but the current API endpoint only takes in those 3, right? The labels are only setup for the 3 params. Should I add the new parameter now itself?

Reply:
For the new API endpoint functionality as in https://github.com/TEAMMATES/teammates/pull/10984, I've recently changed it to take in `fsname` too! Sorry if it's a source of confusion.